runtime/kusama/src/lib.rs

```diff
@@ -1481,8 +1481,9 @@ impl Get<&'static str> for StakingMigrationV11OldPallet {
 }
 
 pub type Migrations = (
+	pallet_balances::migration::ResetInactive<Runtime>,
 	pallet_balances::migration::MigrateToTrackInactive<Runtime, CheckAccount>,
```

Comment:
```suggestion
	// We need to apply this migration again, because `ResetInactive` resets the state again.
	pallet_balances::migration::MigrateToTrackInactive<Runtime, CheckAccount>,
```

We should leave some comment here for the release team.